config/application.yml.example

```diff
@@ -103,30 +103,35 @@ idsr_sms:
 #   username: admin
 #   password: test
 
+
 #To get the sms key follow the following steps in the ReadMe
-development:
-  sms_gateway_url: ''
-  sms_api_key: ''
-  sms_reminder: 'true'
-  next_appointment_message: 'Okondewa  dziwani kuti mukuyera kukabayisa katemera'
-  next_appointment_reminder_period: 'Instant reminder'
-  sms_activation: 'true'
-  show_sms_popup: 'true'
-
-test:
-  sms_gateway_url: ''
-  sms_api_key: ''
-  sms_reminder: 'true'
-  next_appointment_message: 'Okondewa  dziwani kuti mukuyera kukabayisa katemera'
-  next_appointment_reminder_period: 'Instant reminder'
-  sms_activation: 'true'
-  show_sms_popup: 'true'
+eir_sms_configurations:
+  development:
+    sms_gateway_url: ''
+    sms_api_key: ''
+    sms_reminder: true
```

Comment:
Being that we have put these in site based config let us remove them from this config